legacy/README.md

```diff
@@ -94,11 +94,14 @@ please contact us.
 | Fedora     | x86_64 | 35    | GCC 11.3.1                 | 3.24.2 |
 | Fedora     | x86_64 | 36    | GCC 12.2.1                 | 3.24.2 |
 | Fedora     | x86_64 | 37    | GCC 12.2.1                 | 3.24.2 |
+| Fedora     | x86_64 | 41    | GCC 14.2.1                 | 3.30.5 |
 | macOS      | x86_64 | 12    | AppleClang 13, gfortran 12 | 3.24.1 (brew) |
 | macOS      | x86_64 | 11    | AppleClang 13, gfortran 12 | 3.24.1 (brew) |
 | macOS      | arm64  | 12    | AppleClang 13, gfortran 12 | 3.24.1 (brew) |
+| OpenSuSe   | x86_64 | 15.5  | GCC 7.5.0                  | 3.20.4 |
 | Ubuntu     | x86_64 | 20.04 | GCC 9.4.0                  | 3.24.3 (`bootstrap-cmake.sh`) |
 | Ubuntu     | x86_64 | 22.04 | GCC 11.3.0                 | 3.22.1 |
+| Ubuntu     | x86_64 | 24.04 | GCC 13.2.0                 | 3.28.3 |
```

Comment:
If you claim, that those are tested, then please
1. Add them to the container build rules: https://github.com/FairRootGroup/FairSoft/tree/master/test/container/legacy
2. Ping @dennisklein to rebuild the CI containers
3. Add them to Jenkinsfile for actual testing
4. Then add to README.

I'd suggest to perform all that in another PR round.

Reply:
I overlooked the the line claiming that the  systems are tested regularly. This is for sure not the case. They were only tested once. I agree to remove them from this list, but there is still the question if we want to add information about systems which were only tested once.

Reply:
Now I wonder. Looking at the build and test matrix for this PR I see that only 4 systems are tested. How do we test the rest of the list regularly?

Reply:
As already explained privately, we used to have different test sets for Pull Requests and for "branch builds". So when master or any branch gets an update, a bigger test suite is running.

The idea is to have "faster responses" on PRs and the full coverage on the main braanches.

Github lately introduced a "patch queue", which we could use for a middle thing: Let the initial testing for a PR be quick. When we consider the thing to be "good enough", we can press the "add to merge queue" and then a full test round is running. If it is successful, it's merged and done. Otherwise it's back to square one.

Just ponedering. As also already said privately–I am already quite far away from all of this.

Reply:
I have added a new section about systems not yet in the test matrix but tested manually.